### ISSUE TITLE
Being Run Over By Driven Vehicle During Portal Storm

### ISSUE THREAD

**Issue by Kiael** [CleverRaven/Cataclysm-DDA#60096]
### Describe the bug

While driving in an APC with an intact seatbelt and seat with a heavy duty frame behind me, I have twice had my character randomly slam into the frame without crashing into anything, pulling a handbreak, or changing speed. All vehicle parts are completely intact when examined after death (I had an NPC that I switched over to when I died). 

### Steps to reproduce

I have tried to reproduce this over and over again and have been unable to, there doesn't seem to be any rhyme or reason as to why it happens when it does. The only common denominator was the empty heavy duty frame directly behind my character, and on one of the occasions I was using a seatbelt instead of a five point harness if that changes anything.

### Expected behavior

To not be run over while driving the car.

### Screenshots

![unknown](https://user-images.githubusercontent.com/111107272/184289308-358523bb-e9ff-4528-8c83-f48b95143af8.png)
![image](https://user-images.githubusercontent.com/111107272/184289377-2beca877-c97a-4289-a5be-4065e9a9a6a9.png)

### Versions and configuration

Windows 10
Build 2022-08-11-0819 (This happened on 08/09/2022 as well but I can't remember which build exactly it was)
Mods: No fungal growth, bionic professions, no NPC needs

### Additional context

Without changing anything about my vehicle, I reloaded an old backup and crashed it into multiple trees, multiple times, and the full frontal 40 mph force of those impacts did not kill my character, while this did, making me pretty sure it wasn't caused by a collision of any kind with anything on the map. In the images provided there is a collision heard but only after the damage is inflicted, and on investigation no debris was left beside my character's corpse. I would provide a save but I alt f4'd after I gathered the information that I did so I wouldn't lose my character to whatever issue this is.

**Comment by EIIKaO** [comment]
IMO, surveyed your screenshots, I strongly suspected that the portal storms(effect) was the cause(in which case it could be the specification, not a bug).
However I'm not at all familiar with current portal storms, would like to wait for other experts opinions.

**Comment by Kiael** [comment]
> IMO, surveyed your screenshots, I strongly suspected that the portal storms(effect) was the cause(in which case it could be the specification, not a bug). However I'm not at all familiar with current portal storms, would like to wait for other experts opinions.

There was no portal storm when it happened the first time, which makes me think if it is related its only tangential.

**Comment by PatrikLundell** [comment]
Portal storms is a known cause of this, because the thick time thing doesn't work properly, and there may be other causes as well (I wouldn't be shocked if the sprouting of alien grass under a moving vehicle could cause it, for instance).

Another known case of not being registered as being in the vehicle's driver's seat (and probably on the ground in the same tile) while in a vehicle, that I think has been fixed, was when handing in quests at the Tacoma farm. That wasn't harmful as the vehicle was stationary, though.

I also think being tossed (by a monster) while in a vehicle can cause you to end up under it rather than inside on another tile.

I don't think the property of the tile behind is significant, but that this time was caused by the portal storm, while the previous instance was caused by something else.

**Comment by andrei8l** [comment]
A save file from right before/after would be useful here, even if everything looks and works fine after reloading.

> Portal storms is a known cause of this

Can you link another issue for this? AFAIK, I've fixed the known causes, but I haven't been playing close attention lately.

**Comment by PatrikLundell** [comment]
No, I wasn't aware of the portal storm issue being fixed, so it seems I'm using old data here...

**Comment by anoobindisguise** [comment]
I've had the issue of portal storms randomly killing me instantly while driving even when wearing the 5 point anchor, but I've driven considerably unsafe cars fairly regularly and never had this happen without a portal storm being active.

**Comment by Kiael** [comment]
> A save file from right before/after would be useful here, even if everything looks and works fine after reloading.
> 
> > Portal storms is a known cause of this
> 
> Can you link another issue for this? AFAIK, I've fixed the known causes, but I haven't been paying close attention lately.

The closest save that I have is this one. I loaded the previous autosave that was taken 5 hours before the death during the portal storm and added a cloth board to the heavy duty frame, and then made this backup (added the cloth board so my character wouldn't randomly instadie in the backup but I now realize after reading through most of the responses the heavy duty frame almost certainly has nothing to do with what happened)
[Testing.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/9329879/Testing.zip)
.

**Comment by andrei8l** [comment]
/confirmed Caused by the long-range teleport hack used in `map::transform_radius()`

most easily triggered by running `EOC_PORTAL_GRASS_CHANGE` then shifting the map until it's out of the reality bubble and waiting 30 minutes
